playbooks/aws/openshift-cluster/launch.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Launch instance(s)
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars_files:
+      - vars.yml
+  tasks:
+    - set_fact: k8s_type="master"
+
+    - name: Generate master instance names(s)
+      set_fact: scratch={{ cluster_id }}-{{ k8s_type }}-{{ '%05x' |format( 1048576 |random) }}
+      register: master_names_output
+      with_sequence: start=1 end={{ masters }}
```

Comment:
I assume masters is a number by how it's used. Should we call it num_masters instead?

I'm not sure where this name is being set. To be clear, not a blocker to merge.